documentation/modules/mtv-overview-page.adoc

```diff
@@ -6,63 +6,113 @@
 [id="mtv-overview-page_{context}"]
 = The MTV Overview page
 
-The {project-first} *Overview* page displays system-wide information about migrations and a list of *Settings* you can change.
+If you have Administrator privileges, you can access the *Overview* page by clicking *Migration for Virtualization > Overview* in the {ocp} web console.
 
-If you have Administrator privileges, you can access the *Overview* page by clicking *Migration* -> *Overview* in the {ocp} web console.
-
-The *Overview* page has 3 tabs:
+The *Overview* page has 4 tabs:
 
 * Overview
 * YAML
-* Metrics
+* Health
+* Settings
 
 [id="overview-tab_{context}"]
 == Overview tab
 
-The *Overview* tab lets you see:
+The *Overview* tab is designed to let you create providers quickly and to make it easier for you to find system-wide information:
 
-* Operator: The namespace on which the {project-short} Operator is deployed and the status of the Operator
-* Pods: The name, status, and creation time of each pod that was deployed by the {project-short} Operator
-* Conditions: Status of the {project-short} Operator:
+* In the upper-right pane, following the *Welcome* message, are buttons that let you open the *Create provider* UI for each vendor ({vmw}, Open Virtual Appliance, OpenStack, {rhv-full}, and {virt}).
 
-** Failure: Last failure. `False` indicates no failure since deployment.
-** Running: Whether the Operator is currently running and waiting for the next reconciliation.
-** Successful: Last successful reconciliation.
+//Confirm time and list or kebab
+* In the center-left pane is a "donut" chart named *Virtual machines*. This chart shows the number of running, failed, and succeeded virtual machine migrations that {project-short} ran for the time interval that you select. You can choose a different interval by clicking the list in the upper-right corner of the pane. You can choose a different interval by clicking the list. Options: 7 days, 31 days, 24 hours.
+
+* In the center-right pane is an area chart named *Migration history*. This chart shows the number of migrations that succeeded, failed, or were running during the interval shown in the title of the chart. You can choose a different interval by clicking the {kebab} in the upper-right corner of the pane. Options: 7 days, 31 days, 24 hours.
+
+//Confirm time and list or kebab
+* In the lower-left pane is a "donut" chart named *Migration plans*. By default, this chart shows the number of running, failed, succeeded, or canceled migration plans that {project-short} ran for the time interval that you select. You can choose a different interval by clicking the list in the upper-right corner of the pane. List options are: 7 days, 31 days, 24 hours. You can also see the number of plans that were not started, cannot be started, are incomplete, archived, paused, or have an unknown status by clicking the *Show all plans* link.
```

Comment:
no range selector for Migration plans donut chart. It always gives you the current summary of plan statuses